src/app/pages/DashboardPage/SnapshotCard.tsx

```diff
@@ -8,19 +8,20 @@ import { styled } from '@mui/material/styles'
 import Typography from '@mui/material/Typography'
 
 export const StyledCard = styled(Card)(({ theme }) => ({
+  display: 'flex',
   flex: 1,
-  // used to create a stronger selector to override default theme styles
-  [theme.breakpoints.down('sm')]: {
-    padding: 0,
-  },
-  [theme.breakpoints.up('sm')]: {
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  '&': {
     padding: 0,
   },
+  height: 186,
 }))
 
 const StyledCardContent = styled(CardContent)(({ theme }) => ({
   position: 'relative',
   paddingTop: theme.spacing(4),
+  height: 90,
```

Comment:
Do we need it as we already set height of a card in L~18?

Reply:
After removing the aspect ratio, so that the height stays consistent, this is needed here.

Reply:
The issue was, when changing screen size, aspect ration kept the height in sync of the aspect ration resulting in overflowing the container(which had fixed size).